src/osip-core/src/main/java/edu/kit/pse/osip/core/io/files/BaseParser.java

```diff
@@ -37,6 +39,9 @@ protected final void die(String message) throws ParserException {
      */
     protected final char pop() throws ParserException {
         if (available()) {
+            if (toParse.charAt(currentPosition) == '\n') {
```

Comment:
Bessere Fehlermeldungen. Wenn der Parser abbricht, kann er jetzt nicht nur den Character anzeigen, in dem das Problem aufgetreten ist. Er kann die Zeile direkt anzeigen